sources/packages/web/src/assets/css/base.scss

```diff
@@ -219,3 +219,15 @@ ul.no-bullets {
   max-height: none !important;
   max-width: none !important;
 }
+
+.v-alert-background-success {
```

Comment:
Is it not possible to make it be selected by the classes already present on the alert component?